renderdoc/driver/d3d11/d3d11_device.cpp

```diff
@@ -2934,7 +2934,7 @@ bool WrappedID3D11Device::Serialise_SetShaderExtUAV(SerialiserType &ser, GPUVend
       }
       m_ReplayNVAPI->SetShaderExtUAV(~0U, reg, true);
     }
-    else if(vendor == GPUVendor::AMD)
+    else if((vendor == GPUVendor::AMD) || (vendor == GPUVendor::Samsung))
```

Comment:
I've worked in environments where you are expected not to rely on built-in precedent in compound expressions; you must be explicit using parentheses. The thinking is that many a subtle bug has occurred in software development because of a misunderstanding/misremembering of the precedent rules. But I'm happy to oblige with your preference, of course.

Reply:
Yeh I certainly use it myself for complex expressions, or expressions with operators in the same class of logical, arithmetic, etc, where the precedence rules are well defined but not always immediately clear so parentheses make it safer and less error prone. E.g. I would personally prefer `(a && b) || c` over `a && b || c` even if they are equivalent, but I consider it safe to treat simple comparisons as unambiguous. Adding too many explicit parentheses can reduce readability as well.

Taken to the extreme, the `->` operator is also an operator with a precedence, but I wouldn't want to do something like `(ptr->member) == (otherptr->member)` :grinning:.